tests/postgres/vm.test.ts

```diff
@@ -939,3 +943,310 @@ test('onSnapshot', async () => {
     await vm.methods.onSnapshot!(event)
     expect(fetch).toHaveBeenCalledWith('https://google.com/results.json?query=$snapshot')
 })
+
+describe('exportEvents', () => {
+    test('normal operation', async () => {
+        const indexJs = `
+            async function exportEvents (events, meta) {
+                await fetch('https://export.com/results.json?query=' + events[0].event + '&events=' + events.length)
+            }
+        `
+        await resetTestDatabase(indexJs)
+        const vm = await createPluginConfigVM(
+            hub,
+            {
+                ...pluginConfig39,
+                config: {
+                    ...pluginConfig39.config,
+                    exportEventsBufferBytes: 10000,
+                    exportEventsBufferSeconds: 1,
```

Comment:
think this has type string above
```suggestion
                    exportEventsBufferSeconds: '1',
```

Reply:
makes sense, fixed